addons/project/report/project_report.py

```diff
@@ -52,6 +52,10 @@ class ReportProjectTaskUser(models.Model):
         column1='project_task_id', column2='project_tags_id',
         string='Tags', readonly=True)
     parent_id = fields.Many2one('project.task', string='Parent Task', readonly=True)
+    rating_last_text = fields.Selection(string="Rating Last Text", related="task_id.rating_last_text")
```

Comment:
To avoid trigger the report when a rating is added to the task, we should create a compute without any dependencies and a method search. Because related field create automatically a compute and search, and the compute method has a dependency to be retrigger when the `rating_last_text` change in the task.
```suggestion
    rating_last_text = fields.Selection(string="Rating Last Text", compute="_compute_rating_last_text", search="_search_rating_last_value")
```

And then before the `_search` method we will add the both methods:
```py
def _compute_rating_last_text(self):
    for task_analysis in self:
        task_analysis.rating_last_text = task_analysis.task_id.rating_last_text

def _search_rating_last_text(self, operator, value):
    return [('task_id.rating_last_text', operator, value)]
```